### PR TITLE
Feature/obasoro kunle

### PR DESCRIPTION
**Pull request by Obasoro**
## Description

- Add Obasoro kunle to the contibutors list

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by CBID2 — CHANGES_REQUESTED** [review]
Thanks for your Pull Request @Obasoro! We've merged in some new PRs since you submitted. Can you merge the latest changes and resolve conflicts? If you need some help with that, check out this post: [Keeping Your Branch Up to Date and Handling Merge Conflicts While Waiting for PR Reviews](https://dev.to/opensauced/keeping-your-branch-up-to-date-and-handling-merge-conflicts-while-waiting-for-pr-reviews-3b3h).

**Comment by Obasoro** [comment]
@CBID2 Thanks for the message. Just for better understanding `Can you merge the latest changes and resolve conflicts?`, would need some clarity on this.

**Comment by CBID2** [comment]
> @CBID2 Thanks for the message. Just for better understanding `Can you merge the latest changes and resolve conflicts?`, would need some clarity on this.

We mean to update your branch, but it looks like you did a force push instead.

**Comment by Obasoro** [comment]
Thanks @CBID2. On mt `feature/obasoro-kunle` showing `This branch is not behind the upstream open-sauced:main
`. Trying to understand the next action.

**Comment by CBID2** [comment]
> Thanks @CBID2. On mt `feature/obasoro-kunle` showing `This branch is not behind the upstream open-sauced:main `. Trying to understand the next action.

Did my suggestion work @Obasoro?

**Comment by Obasoro** [comment]
Thanks @CBID2 . Sometimes I don't want to get things messed up. Still learning the ropes and #Opensauce is place to get my skill sharpen and also my hands dirty

**Comment by CBID2** [comment]
> Thanks @CBID2 . Sometimes I don't want to get things messed up. Still learning the ropes and #Opensauce is place to get my skill sharpen and also my hands dirty

I updated my previous comment @Obasoro. Check it out.

**Comment by Obasoro** [comment]
Do have to submit another pull request?

**Comment by CBID2** [comment]
> Do have to submit another pull request?

Let's wait for Bekah's response first.

File: README.md
Comment (adiati98) [inline README.md:206]: This should be generated once after you run the `npm run contributors:generate`.
But there is a duplication below.

Delete them both and run `npm run contributors:generate` after you added yourself in `.all-contributorsrc`.

File: README.md
Comment (adiati98) [inline README.md:215]: The duplication.

File: README.md
Comment (adiati98) [inline README.md:131]: We don't want to modify this, but there's badge duplication here.
So, delete line 126-128 and 130.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @Obasoro,

I see that you haven't update your forked repo. That's why you still have conflicts.
Click the "Sync fork", then "Update branch". Next, update your local working branch.

I left you some comments here. After you updated your local working branch, please fix them first before we can merge your PR.

If you have questions do let us know :)

**Comment by Obasoro** [comment]
Added Obasoro Kunle manually.

**Comment by CBID2** [comment]
There's still merge conflicts on this branch @Obasoro. Have considered trying this method: https://blog.kaleidoscope.app/2022/04/12/resolving-merge-conflicts-in-vscode-with-kaleidoscope/

**Comment by adiati98** [comment]
@Obasoro, is there any reason you close the PR and force-push your commits?

**Comment by Obasoro** [comment]
@adiati98 I added everything back and thought everything was going work fine but failed so had to close it and create a new one.

**Pull request closed without merging**
